src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -333,135 +302,60 @@ private synchronized void syncAcl() {
-                esClient.admin().indices().create( new CreateIndexRequest().index(searchGuardIndex)
-                        .settings("index.number_of_shards", 1, "index.number_of_replicas", chr.getNumberOfDataNodes()-1)
-                        ).actionGet().isAcknowledged();
-            }
-
-			// Wait for health status of YELLOW
-			ClusterHealthRequest healthRequest = new ClusterHealthRequest()
-					.indices(new String[]{searchGuardIndex}).waitForYellowStatus();
-
-			esClient.admin().cluster().health(healthRequest).actionGet().getStatus();
-
-			String cd = settings.get(SG_CONFIG_SETTING_PATH, DEFAULT_SG_CONFIG_SETTING_PATH);
-			// create initial sg config
-			for ( String type : SEARCHGUARD_INITIAL_CONFIGS ) {
-				String file = String.format("%ssg_%s.yml", cd, type);
-				logger.debug("Using '{}' file to populate '{}' type", file, type);
-				if ( ! uploadFile(esClient, file, type) ) {
-					String failedSeed = String.format("'%s/%s/%s' with '%s'", searchGuardIndex, type, SEARCHGUARD_CONFIG_ID, file);
-					logger.error("Was not able to seed {}", failedSeed);
-				}
-			}
-
-            ConfigUpdateResponse cur = esClient.execute(ConfigUpdateAction.INSTANCE, new ConfigUpdateRequest(SEARCHGUARD_INITIAL_CONFIGS)).actionGet();
-            logger.debug("Succeed seeding SG, all node's config updated? '{}'", cur.getNodes().length == chr.getNumberOfNodes());
-        }
-	}
-
-	private void seedInitialACL(Client esClient) throws Exception {
-
-		SearchGuardRoles roles = new SearchGuardRoles();
-		SearchGuardRolesMapping rolesMapping = new SearchGuardRolesMapping();
-		try {
-			if ( doesSearchGuardIndexExist()) {
-
-				//This should return nothing initially - if it does, we're done
-				roles = readRolesACL(esClient);
-				rolesMapping = readRolesMappingACL(esClient);
-
-				if ( roles.iterator().hasNext() || rolesMapping.iterator().hasNext() ) {
-					if ( logger.isDebugEnabled() )
-						logger.debug("Have already seeded");
-					seeded = true;
-					return;
-				}
-			}
-		}
-		catch (NoNodeAvailableException | NoShardAvailableActionException e) {
-			logger.warn("Trying to seed ACL when ES has not not yet started: '{}'", e.getMessage());
-			return;
-		}
-		catch (IndexNotFoundException | DocumentMissingException | NullPointerException e) {
-			logger.debug("Caught Exception, ACL has not been seeded yet", e);
-		}
-		catch (Exception e) {
-			logger.error("Error checking ACL when seeding", e);
-			throw e;
-		}
-
-		try {
-			// seed here from the files
-			createConfig(esClient);
-		}
-		catch (Exception e) {
-			logger.error("Error seeding initial ACL", e);
-			throw e;
-		}
-
-		seeded = true;
-	}
-	
-	private boolean doesSearchGuardIndexExist() {
-	    return esClient.admin().indices().exists(new IndicesExistsRequest().indices(new String[]{searchGuardIndex})).actionGet().isExists();
+	    IndexRequest rolesIR = new IndexRequest(searchGuardIndex).type(SEARCHGUARD_ROLE_TYPE).id(SEARCHGUARD_CONFIG_ID).refresh(true)
```

Comment:
its hard to tell what changed between these two sections, it looks like there was some autoformatting